js/card.js

```diff
@@ -4,8 +4,22 @@ window.card = (function () {
   var cardTemplate = document.querySelector('#card').content;
   var map = document.querySelector('.map');
   var mapFiltersContainer = map.querySelector('.map__filters-container');
+  var mapPins = map.querySelector('.map__pins');
+
+  function handlePopupEsc(evt) {
+    if (evt.key === 'Escape') {
+      removeCard();
+      document.removeEventListener('keydown', handlePopupEsc);
+    }
+  }
 
   function renderCard(add) {
+    var TYPE_OF_HOUSE = {
```

Comment:
чуть по-другому, как у встроенной константы Math.PI
```suggestion
    var type_of_house = {FLAT: ...}
```